server/node-postgres-mydb/db/seeds/dev/03_band_has_artist.js

```diff
@@ -0,0 +1,19 @@
+const songs = require(' ../../../seed.json');
```

Comment:
You don't actually use any data from the json file in this file?

Reply:
yes I getting empty record and I dont know why i am not accessing the data